core/server/middleware/index.js

```diff
@@ -171,10 +170,6 @@ setupMiddleware = function setupMiddleware(blogApp) {
     // must happen AFTER asset loading and BEFORE routing
     blogApp.use(prettyURLs);
 
-    // Body parsing
```

Comment:
If we created an app that did that, then I would expect that app to be in charge of determining the body parser settings, rather than Ghost core.